src/app/components/challenge-create/challenge-create.component.scss

```diff
@@ -65,7 +66,7 @@
 /*media queries*/
 
 @media only screen and (max-width: $med-screen) {
-  .web-container {
+  .challenge-create-flex {
```

Comment:
Please revert this change too. (it should set the width to 100% of web-container)

Reply:
Done @Sanji515